demisto_sdk/commands/common/content/objects/pack_objects/xsiam_report/xsiam_report.py

```diff
@@ -30,3 +31,10 @@ def upload(self, client: demisto_client):
 
     def type(self):
         return FileType.XSIAM_REPORT
+
+    def dump(self, dest_dir: Optional[Union[Path, str]] = None) -> List[Path]:
+        created_files: List[Path] = []
+        created_files.extend(super().dump(dest_dir=dest_dir))
+        new_file_path = str(created_files[0])
+        shutil.copyfile(new_file_path, new_file_path.replace('external-', ''))
+        return super().dump(dest_dir)
```

Comment:
Why are you calling `super().dump` twice? (same for all)

Reply:
I fixed it